docs/source/index.rst

```diff
@@ -55,7 +53,7 @@ Documentation
       :text: Developer Guide
       :classes: btn-link stretched-link font-weight-bold
   
-  adding new functionality to DIRAC
+  adding new functionalities to DIRAC
```

Comment:
Never trust a native speaker (well, never trust this one) but I think this was already correct:
```suggestion
  adding new functionality to DIRAC
```

Reply:
really? Well I can trust you better than myself, of course, at least on this point. I correct it back.